lib/generator.js

```diff
@@ -349,7 +349,15 @@ const generateMd = (inventory, options = {}) => {
             )
           )
         );
-      } else if (method.indexOf('.') !== -1) {
+      } else if (
+        method.indexOf('.') !== -1 &&
+        methods.has(
+          method
+            .split('.')
+            .slice(0, -1)
+            .join('.')
```

Comment:
`method.substring(0, method.lastIndexOf('.'))`?
Also, I think this be better extracted in a variable.